app/src/test/java/com/codesoom/assignment/applications/ProductServiceTest.java

```diff
@@ -0,0 +1,183 @@
+package com.codesoom.assignment.applications;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import com.codesoom.assignment.dto.ProductNotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("ProductServiceTest 클래스")
+class ProductServiceTest {
+    private ProductService productService;
+
+    private ProductRepository productRepository;
+
+    private Product product1;
+    private Product product2;
+
+    @BeforeEach
+    public void setUp() {
+
+        product1 = new Product(1L, "toy1", "maker1", 1000L, "toy1.jpg");
+        product2 = new Product(2L, "toy2", "maker2", 2000L, "toy2.jpg");
+
+        productRepository = new ProductRepository();
+        productService = new ProductService(productRepository);
+    }
+
+    @Nested
+    @DisplayName("save 메소드")
+    class Describe_save {
+
+        @Test
+        @DisplayName("새로운 product를 반환합니다.")
+        void it_return_new_product() {
+            assertThat(productService.save(product1)).isEqualTo(product1);
```

Comment:
이렇게 해보면 어떨까요? `새로운`이 `isEqualTo`와 잘 안 맞는 표현인 것 같아요.
```suggestion
        @DisplayName("요청한 내용으로 저장된 product를 반환합니다.")
        void it_return_new_product() {
            assertThat(productService.save(product1)).isEqualTo(product1);
```

Reply:
말씀해주신 표현이 좀더 구체적이고 명확한 표현이네요.